xarm_moveit_config/config/xarm6/controllers.yaml

```diff
@@ -6,9 +6,9 @@ xarm6_traj_controller:
   type: FollowJointTrajectory
   default: true
   joints:
-    - joint1
-    - joint2
-    - joint3
-    - joint4
-    - joint5
-    - joint6
+    - xarm6_joint1
```

Comment:
This has to match URDF joint names (after xacro substitution). Move to description or similar package in the future.